src/plugins/navigation_embeddable/public/editor/open_editor_flyout.tsx

```diff
@@ -79,8 +77,6 @@ export async function openEditorFlyout(
     );
 
     editorFlyout.onClose.then(() => {
-      // we should always re-fetch the dashboards when the editor is opened; so, clear the cache on close
-      memoizedFetchDashboards.cache = new memoize.Cache();
       closed$.next(true);
```

Comment:
Now that the dashboard content management service is handling its **own** cache, we don't need to worry about this :+1: